linux_os/guide/system/accounts/accounts-restrictions/root_logins/no_shelllogin_for_systemaccounts/oval/shared.xml

```diff
@@ -93,7 +93,7 @@
   <!-- Thus check that all /etc/passwd entries having shell defined
        have UID outside of <0, UID_MIN -1> range -->
   <ind:textfilecontent54_test state_operator="OR" id="test_shell_defined_default_uid_range" check="all"
-  check_existence="all_exist" comment="&lt;0, UID_MIN - 1&gt; system UIDs having shell set"
+  check_existence="any_exist" comment="&lt;0, UID_MIN - 1&gt; system UIDs having shell set"
```

Comment:
Nice catch, do think it would make sense to change the comment above to:
 `<!-- Thus check that any /etc/passwd entry with shell defined`

If you agree, would you mind doing that for `test_shell_defined_reserved_uid_range` and `test_shell_defined_dynalloc_uid_range`?

Reply:
Sure, no problem. Will add the comment to the other tests as well.